### ISSUE TITLE
Italics does not render in Firemonkey when SKIA is enabled on Apple platforms iOS and MACOS

### ISSUE THREAD

**Issue by jareddavison** [skia4delphi/skia4delphi#322]
Italics do not render in Firemonkey app when SKIA is enabled on Apple platforms iOS and MACOS

I've logged this issue also with Embarcardero as [RSS-1231](https://embt.atlassian.net/servicedesk/customer/portal/1/RSS-1231)

**Comment by viniciusfbb** [comment]
@jareddavison I suspect that the limitation is related only to Oblique font slant, not Italic. I am now without my mac to test it. Can you test it?
You can edit the .fmx file directly in the notepad. The follow line is for FontSlant=Italic:
```delphi
Font.StyleExt = {02040000000100000004000000}
```
The follow line is for FontSlant=Oblique:
```delphi
Font.StyleExt = {02040000000200000004000000}
```

**Off-topic:** Just to who it may interest, I also made a tool to convert font weight, slant and stretch to TFontStyleExt and vice-versa as you cannot choose it using the IDE UI on the previous versions of RAD Studio:
[Font.StyleExt.zip](https://github.com/user-attachments/files/16045971/Font.StyleExt.zip)

**Comment by jareddavison** [comment]
Thank you @viniciusfbb . that's interesting but I don't really understand the implication. Is this a style mapping issue?

See the design time (Windows) and runtime (MacOS) versions of the form:

![FontExtStyle](https://github.com/skia4delphi/skia4delphi/assets/11552748/88c369f3-d2a4-4613-9726-8f5a0b52ece2)

Updated project files:
[SkiaItalicsTest2.zip](https://github.com/user-attachments/files/16046462/SkiaItalicsTest2.zip)

**Comment by jareddavison** [comment]
For interest's sake I thought I'd also show there is a slight difference between the design time and runtime on windows. I suspect this is because design time is not using SKIA where runtime is, am I right?
![FontExtStyle-Windows-SKIA](https://github.com/skia4delphi/skia4delphi/assets/11552748/163020f6-6cec-46b9-8635-e003ab85953a)

**Comment by viniciusfbb** [comment]
@jareddavison I don't know exactly the reason for this yet, and I really want to investigate more about this first. However, now we already have a possible workaround:
https://github.com/skia4delphi/skia4delphi/blob/0a62f87efd325440323aaaabed93eec4c00b6c4f/Source/FMX/FMX.Skia.Canvas.pas#L3033
to:
```delphi
  SkFontSlant : array[TFontSlant] of TSkFontSlant = (TSkFontSlant.Upright, TSkFontSlant.Italic, {$IFDEF MACOS}TSkFontSlant.Italic{$ELSE}TSkFontSlant.Oblique{$ENDIF});
```

**Comment by viniciusfbb** [comment]
> I suspect this is because design time is not using SKIA where runtime is, am I right?

Yes. Design-time does not use skia because the GlobalUseSkia is False by default and your application change it only in runtime.

**Comment by jareddavison** [comment]
Here's the results of the patch on MacOS.
![Uploading FontExtStyle-MacOS-SKIA-patched.jpg…]()

**Comment by jareddavison** [comment]
Also, here's the results for iPhone simulator:
![FontExtStyle-iOS-Simulator-SKIA-patched](https://github.com/skia4delphi/skia4delphi/assets/11552748/e0171fa2-2b69-4318-a67b-275030137959)

**Comment by viniciusfbb** [comment]
@jareddavison The user Hans Jakobsen said on [RSS-1231](https://embt.atlassian.net/servicedesk/customer/portal/1/RSS-1231) that Italic + Bold does not work. Can you confirm if that workaround works for italic + bold also?

**Comment by jareddavison** [comment]
Yes I can. I will reply in a day or two.

**Comment by jareddavison** [comment]
I couldn't reproduce it the report. See image below. Using the unpatched FMX.Skia.Canvas, bold does appear bold, but Italics was broken.

![image](https://github.com/user-attachments/assets/25ceac5d-2d3e-4d47-ae43-3207a8f3748a)

**Comment by viniciusfbb** [comment]
@jareddavison Thank you very much!

**Comment by viniciusfbb** [comment]
Fixed on latest release **[v6.3.0-beta1](https://github.com/skia4delphi/skia4delphi/releases/tag/v6.3.0-beta1)**.
